service/auth/jwt/jwtConfig.go

```diff
@@ -0,0 +1,39 @@
+package jwt
+
+import (
+	"errors"
+	"os"
+	"strconv"
+)
+
+type Config struct {
+	pathCert            string
+	certVersion         string
+	tokenExpireDuration int
+}
+
+func NewJTWConfig() (*Config, error) {
+	certVersion := os.Getenv("CERT_VERSION")
```

Comment:
You should read all the env variables in one place, typically in cmd/<name> and later add it to functions as a parameter.

Reply:
Config is used to change variables in runtime(not only when config server) to use dynamic variables(for example change cert during work).

Reply:
Also this relies on [prev comment](https://github.com/stasBigunenko/monorepa/pull/25#discussion_r708090331) and will be fixed if needed together.